lib/rbi/rewriters/remove_known_definitions.rb

```diff
@@ -81,8 +81,6 @@ def visit_all(nodes)
 
       sig { override.params(node: T.nilable(Node)).void }
       def visit(node)
-        return unless node
```

Comment:
I'm not convinced removing these is a great idea. They're _technically_ redundant, but they also fail fast on `nil`, which would be a really common case to hit (rather than doing 3 `Class#===` checks below)